code-challenges.js

```diff
@@ -21,29 +21,133 @@ const temp1 = 42
+
+
+
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
-const currentCohort = "Bravo 2022"
+// Psuedo Coding
+// Create a code that will reverse the letters of a string
+// I decleared I will reverse my variable of bravoCohort declared as "Bravo 2022"
+// In order to do that I will first split that will read each letter speratle.
+// Then I will make them reverse in an array by removing the ""
+// Then I will join them with the .join "" back to a string
+// It returned reversed as expected.
+
+
+const bravoCohort = "Bravo 2022"
+
+let reverse = bravoCohort.split("").reverse().join("")
+
+console.log(reverse)
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
+// const myArray = [13, 34, 5, 10, 27, 42]
+
+// Psuedo Coding
+// Create a statement aka loop
+// The statment must evaluate which numbers in the array above classified as an even or odd value using the % 2
+// Use i++ so it will read all numbers continously in the array
+// The expected output should be in the order of "odd" "even" "odd" "even" "odd" "even"
+// After my first try"
+// I had to change my == to === to strictly equal my outcome
+// I had to make sure I read myArray.length to get a number
+// Following the past examples, help and trial I recived the correct outcome.
+
 const myArray = [13, 34, 5, 10, 27, 42]
 
+for (let i=0; i< myArray.length; i++){
+    if (myArray[i] %2 === 0){
+        console.log ("even") ;
+    }else if (myArray[i] %2 === 1){
+        console.log ("odd");
+    }
+}
+
+
+
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
 // Set one:
-const number1 = 58
-const number2 = 100
+// const number1 = 58
+// const number2 = 100
 
 // Set two:
-const number3 = 27
-const number4 = 24
+// const number3 = 27
+// const number4 = 24
+
+// Pseudo Coding
+// Create a function that takes in a set of any two numbers listed below.
+// In the function, you would subtract the smaller number from the larger number. 
+// Each function would need to be addressed with if and else if statements to cover all possibilities and ensure the largest number is subtracting the smallest number.
+// Expected outputs would be 42, 3
+
+// I received some help with this question
+
+// Functions:
+
+const set1 = (num1, num2) => { 
+    if(num1>num2) {
+        return num1-num2
+    }else if(num2>num1){
+        return num2-num1
+    }}
+
+
+console.log(set1 (27,24))
+
+const set2 = (num1, num2) => {
+    if(num1>num2) {
+        return num1-num2
+    }else if(num2>num1){
+        return num2-num1
+    }
+}
+
+console.log(set2 (100, 58))
```

Comment:
Your function set1 was written so wonderfully that there is no need for the second function to be written.  Since set1 is dynamic, allowing 2 parameters to be used throughout the function, all you need to do to reuse that function for both sets of numbers or any set of numbers for that matter would be to pass it as an argument on a different console log.  I also highly recommend using the variables given to test the functions.  Your console logs would look something like this:
`console.log(set1(number1, number2))`
`console.log(set1(number3, number4))`